scripts/build_platform_config.py

```diff
@@ -266,7 +268,7 @@ def codeOutDevicePin(device, pin, definition_name):
   #codeOut("#define JSVAR_CACHE_SIZE                "+str(200)+" // Number of JavaScript variables in RAM")
 else:
   codeOut("#define JSVAR_CACHE_SIZE                "+str(variables)+" // Number of JavaScript variables in RAM")
-  codeOut("#define FLASH_AVAILABLE_FOR_CODE        "+str(flash_available_for_code))
+  codeOut("#define FLASH_AVAILABLE_FOR_CODE        "+str(int(flash_available_for_code)))
```

Comment:
Yes, Python is nice because you don't have to declare variable types. But then again Python is hit because you don't have to declare variable types. Basically, I've installed everything manually, so that's why I have Python 3.

I have a patch for the other variables as well for EFM32, but now I can extend it to the other board as well. 

Should I start pushing to master, or continue to push to efm32integration and issue pull requests?